src/_gettsim/taxes/zu_verst_eink/freibetraege.py

```diff
@@ -332,27 +342,59 @@ def sonderausgaben_betreuung_y_tu(
     return float(out)
 
 
-def eink_st_kinderfreib_y_tu(
-    anz_kinder_mit_kindergeld_tu: float,
-    anz_erwachsene_tu: int,
+def eink_st_kinderfreib_y(
```

Comment:
How do you think about this PR?

1. Proof of concept?
2. Implementation of things doing it the new way?

If 1., we are probably done. If 2., we should strive to get rid of things like `anz_kinder_mit_kindergeld_tu` altogether.

Reply:
I planned to implement the infrastructure in this PR and adjust the existing policy functions in a later PR (that gets rid of `tu` occurrences) such that this PR doesn't get too large. But I'm open to do here too.